src/Nuget/Sarif.Driver.nuspec

```diff
@@ -16,7 +16,7 @@
     <dependencies>
       <group>
         <dependency id="Sarif.Sdk" version="1.4.$patch$-beta" />
-        <dependency id="Newtonsoft.Json" version="7.0.1" />
+        <dependency id="Newtonsoft.Json" version="6.0.0" />
```

Comment:
I believe @version is a package version. So it probably should be 6.0.8. It doesn't look like there's a package version 6.0.0 for Newtonsoft.Json available on NuGet. 

Same for the other nuspec file.